.husky/pre-commit

```diff
@@ -1,6 +1,6 @@
-#!/bin/sh
+#!/usr/bin/env sh
 [ -n "$CI" ] && exit 0
 
-. "$(dirname "$0")/_/husky.sh"
+. "$(dirname -- "$0")/_/husky.sh"
```

Comment:
Copy from one in the `stylelint/stylelint` repo:
https://github.com/stylelint/stylelint/blob/2b4147b8ec5e1aa1136ed858004b53635c022ec3/.husky/pre-commit